src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -106,7 +106,7 @@
                     Visible="False" 
                     ReadOnly="True" />
 
-    <StringProperty Name="TreatSpecificWarningsAsErrors" 
+    <StringProperty Name="WarningsAsErrors" 
```

Comment:
OK, we should update the NuGet spec at https://github.com/NuGet/Home/wiki/Improved-NuGet-warnings to use `<WarningsAsErrors>` in Scenario-1, step 5
/cc anangaur